ArduCopter/GCS_Mavlink.cpp

```diff
@@ -219,6 +219,25 @@ void NOINLINE Copter::send_rangefinder(mavlink_channel_t chan)
             rangefinder.distance_cm_orient(ROTATION_PITCH_270) * 0.01f,
             rangefinder.voltage_mv_orient(ROTATION_PITCH_270) * 0.001f);
 }
+
+void NOINLINE Copter::send_distance_sensor(mavlink_channel_t chan)
+{
+    for (uint8_t i = 0; i < RANGEFINDER_MAX_INSTANCES; i++) {
```

Comment:
I understand this is debatable, but I think this should be doing the same as the rangefinder message - only send the one looking down; when you have others you'll most likely have the proximity feature enabled and the proximity method will already send messages.